### DISCUSSION TITLE
BonVision and DAQmx triggering

### DISCUSSION THREAD

**Discussion by hei-littlemouse** [bonsai-rx/bonsai#1684] [Q&A]
I have two questions regarding the code of BonVision and DAQmx.

1. I am using the code "grating specification"->"CreateGrating Trial"->"Grating Sequence" for the generation of multiple grating in series. The parameters of different grating were set in "grating specification", and it worked well on my first computer. However, when I moved my code to the second computer, the visual stimulation window showed the grating specified in "CreateGrating Trial". I have no idea why the parameters written in "grating specification" do not overwrite those in "CreateGrating Trial". 

2. I have a National Instruments DAQ system for analogue output. May I know how I should code for this purpose? I have the DAQ system working in the MATLAB code so I guess the hardware connection is done. The library of DAQmx was installed already. 

Thank you so much for the help.

**Comment by bruno-f-cruz** [comment]
Heya
1. Include your workflow that minimally reproduces the behavior.
2. [Check the package documentation](https://bonsai-rx.org/daqmx/articles/intro.html)

**Comment by hei-littlemouse** [comment]


**Reply by bruno-f-cruz** [reply]
Include the workflow file as asked in #864. Thanks!
